frontend/src/components/FormItem.js

```diff
@@ -71,12 +76,15 @@ FormItem.propTypes = {
   fieldSetWrapper: PropTypes.bool,
   required: PropTypes.bool,
   className: PropTypes.string,
+  requiredText: PropTypes.string,
 };
 
 FormItem.defaultProps = {
   required: true,
   fieldSetWrapper: false,
   className: '',
+  requiredText: null,
```

Comment:
Nitpicky but In this case I'd probably set the default value to be '(Required)' and drop the conditional on line 50